packages/perseus/src/widgets/radio/__docs__/multiple-choice-demo.new.stories.tsx

```diff
@@ -36,9 +36,9 @@ type StoryArgs = {
  */
 
 export default {
```

Comment:
Now the stories in here will make individual story pages, so that we can easily test different states while developing again.